protocol/proofs/delayed-weth-expansion.md

```diff
@@ -0,0 +1,62 @@
+# Expanded DelayedWETH Capabilities
+
+## Context
+
+The `DelayedWETH` contract acts as a holding contract for the bonded ETH submitted by any
+participant in a `DisputeGame` contract. As of the Granite upgrade, each implementation of the
+`DisputeGame` uses its own `DelayedWETH` contract. Each `DelayedWETH` contract is `Ownable` and is
+subject to a number of safety net actions by the owner address (holding balances from specific
```

Comment:
> As of the Granite upgrade, each implementation of the
`DisputeGame` uses its own `DelayedWETH` contract.

I checked briefly onchain but seems to have only one `DelayedWETH` used currently -> https://etherscan.io/address/0x82511d494B5C942BE57498a70Fdd7184Ee33B975 
Does this means that since Granite each time we `create()` a `DisputeGame` contract,  we also create a `DelayedWETH` contract for the `DisputeGame` instance?

Reply:
We have two `DelayedWETH` contracts ever since Granite. Each dispute game type has its own `DelayedWETH` contract. So every game of the same game type uses the same `DelayedWETH` contract.

Reply:
Got it make more sense to me here!